src/pygeoprocessing/geoprocessing.py

```diff
@@ -1815,6 +1836,9 @@ def warp_raster(
             name string as the first element and a GDAL creation options
             tuple/list as the second. Defaults to a GTiff driver tuple
             defined at geoprocessing.DEFAULT_GTIFF_CREATION_TUPLE_OPTIONS.
+        osr_axis_mapping_strategy (OSR object): The OSR Axis Mapping Strategy 
+            as the first element. Defaults to a GIS friendly Lon,Lat order 
+            defined at geoprocessing.DEFAULT_OSR_AXIS_MAPPING_STRATEGY.
```

Comment:
Could you change the docstring to the one I proposed above?

Reply:
Done in [2eba123](https://github.com/natcap/pygeoprocessing/pull/31/commits/2eba123c585913539377eeaa05ebccdf13990a80)